### PR TITLE
update 2FA language

### PR DESCRIPTION
**Pull request by karann-msft**
stress the need for enabling 2FA and clarify that it does not impact other accounts linked to the MSA

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[be80f1c](https://github.com/NuGet/docs.microsoft.com-nuget/commits/be80f1c7955c6a078c7a8b892686c4b0f3d9b1fc)_: 

:clock10: **Pending**: waiting for processors ([10](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 10 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[be80f1c](https://github.com/NuGet/docs.microsoft.com-nuget/commits/be80f1c7955c6a078c7a8b892686c4b0f3d9b1fc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/nuget-org/individual-accounts.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/karann-2FA/docs/nuget-org/individual-accounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/nuget-org/individual-accounts?branch=pr-en-us-1787) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C20%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201911202052375207-1787%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=KkAY9DR%2BMnX8YA75dXroGolPIlMgz8spy2QXplGtuOs%3D&st=2019-11-20T20%3A48%3A40Z&se=2019-12-21T20%3A53%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
@shishirx34 - I have updated the aka.ms to point to the relevant section - https://aka.ms/nuget-enable-2fa

File: docs/nuget-org/individual-accounts.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ To better protect your account, enable two-factor authentication (recommended).
 
 5. Follow any additional instructions to complete Two-factor authentication.
 
+> [!Tip]
+> Enabling 2FA for your NuGet.org account does not impact authenitcation settings for other accounts or services that may be linked to the Microsoft account you use to login to NuGet.org.
```
Comment (shishirx34) [inline docs/nuget-org/individual-accounts.md]: typo: authentication

Comment (karann-msft) [inline docs/nuget-org/individual-accounts.md]:
```suggestion
> Enabling 2FA for your NuGet.org account does not impact authentication settings for other accounts or services that may be linked to the Microsoft account you use to login to NuGet.org.
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[c7857f6](https://github.com/NuGet/docs.microsoft.com-nuget/commits/c7857f61d15557d9cc0e375035bab16e95e560c5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/nuget-org/individual-accounts.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/karann-2FA/docs/nuget-org/individual-accounts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/nuget-org/individual-accounts?branch=pr-en-us-1787) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C20%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201911202140061370-1787%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2Bmhhd%2BCaC1Wg%2Fkq5HlmwzhkFmAZnbQnT2gUUGoW%2FR8g%3D&st=2019-11-20T21%3A36%3A07Z&se=2019-12-21T21%3A41%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**